target/linux/bcm63xx/dts/bcm6328-adb-pdg-a4101n-a-000-1a1-ae.dts

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "bcm6328.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "ADB P.DG A4101N A-000-1A1-AE";
+	compatible = "adb,pdg-a4101n-a-000-1a1-ae", "brcm,bcm6328";
+
+	aliases {
+		led-boot = &led_dsl_green;
+		led-failsafe = &led_dsl_green;
+		led-running = &led_dsl_green;
+		led-upgrade = &led_dsl_green;
+	};
+
+	chosen {
+		bootargs = "rootfstype=squashfs,jffs2 noinitrd console=ttyS0,115200";
+		stdout-path = "serial0:115200n8";
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		unnamed {
+			label = "unnamed_button";
```

Comment:
What's the stock firmware functionality of this button?
Adding it to OpenWrt as "unnamed_button" doesn't feel right...

Reply:
Unfortunately when I got this router the stock firmware was corrupted and the router didn't boot. Anyway this button is the only one that has no label. In the official user manual this button is not mentioned at all.

I know it doesn't feel right but it's a an unnamed button for real...